File: Daily Test/Java/Split And Find Sum 6.7.2025.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        int sum = 0, i = 0, len = s.length();
        while (i < len) {
            StringBuilder temp = new StringBuilder();
            char cur = s.charAt(i);
            while (i < len && s.charAt(i) == cur) {
                temp.append(s.charAt(i));
                i++;
            }
            sum += Integer.parseInt(temp.toString());
        }
        System.out.println(sum);
        sc.close();
    }
}

File: Daily Test/C/Split And Find Sum 6.7.2025.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main() {
    char s[101];
    scanf("%s", s);
    int sum = 0, i = 0, len = strlen(s);
    while (i < len) {
        char temp[20] = {0};
        int j = 0;
        char cur = s[i];
        while (i < len && s[i] == cur) {
            temp[j++] = s[i++];
        }
        sum += atoi(temp);
    }
    printf("%d\n", sum);
    return 0;
}